### ISSUE TITLE
Fps drops when scrolling chat or opening menu

### ISSUE THREAD

**Issue by AnddyAnddy** [haxball/haxball-issues#2003]
Opera GX, --disable-frame-rate-limit --use-angle=d3d9, Use low latency canvas

Well it starts to be an old issue but it became worse today.
When hovering the mouse with anything related to the UI, the game starts freezing and fps are decreased by a lot.
You can argue it is not an issue when you are playing since you are not scrolling that much the chat or doing actions, but when you are streaming a game, it is awful. I had to go back to normal browser.
I don't know if you can do a lot but before the may updates it was not like that and is fairly annoying
https://imgur.com/a/PVbLNhZ
https://i.imgur.com/MhVchBd.mp4

**Comment by basro** [comment]
I was able to reproduce fps drops when scrolling and opening the menu. Weirdly enough if I open the menu with the Esc key instead of by clicking on the menu button it doesn't seem create any lag. Can you confirm if that's the case for you as well?

It seems the fps drops are caused by mouse interactions only.

I was not able to reproduce creating significant fps drops by just hovering on things however.

I also see that OperaGX no longer works well without the angle d3d9 flag for me. They probably have updated things on their side which may be making things worse.

**Comment by AnddyAnddy** [comment]
> I was able to reproduce fps drops when scrolling and opening the menu. Weirdly enough if I open the menu with the Esc key instead of by clicking on the menu button it doesn't seem create any lag. Can you confirm if that's the case for you as well?

Yes I confirm.

> I was not able to reproduce creating significant fps drops by just hovering on things however.

Huge fps drops are mainly coming from mouse interraction (mousewheel or clicks) or not being focussed on the game window (having 2 screens and using the second for example) but sometimes, just moving the mouse makes it laggy, less than the actions described previously


It became so hard to figure out good settings for haxball it has became a headache to find the best settings, I can't imagine how annoying it would be for new players or players playing without being part of a community.

**Comment by AnddyAnddy** [comment]
I don't think it was the case when I made this post but now it is even lagging when i'm manually setting players teams or even pressing start the game.

**Comment by uzayyli** [comment]
Just today I experienced FPS drop, and I enabled and re-disabled Low Latency Canvas and it restored the FPS. Hope this info helps :p
